resources/cluster-essentials/files/natsclusters.nats.crd.yaml

```diff
@@ -16,20 +16,3 @@ spec:
     singular: natscluster
```

Comment:
Why not move the `NatsCluster` CRD as well?

Reply:
move where?

Reply:
Ah my mistake, it's already in the right place.